typing/ctype.ml

```diff
@@ -2678,14 +2679,16 @@ let complete_type_list ?(allow_absent=false) env fl1 lv2 mty2 fl2 =
   | exception Exit -> raise Not_found
 
 (* raise Not_found rather than Unify if the module types are incompatible *)
-let unify_package env unify_list lv1 p1 fl1 lv2 p2 fl2 =
+let compare_package env unify_list lv1 pack1 lv2 pack2 =
+  let {pack_path = p1; pack_cstrs = fl1} = pack1 in
+  let {pack_path = p2; pack_cstrs = fl2} = pack2 in
   let ntl2 = complete_type_list env fl1 lv2 (Mty_ident p2) fl2
   and ntl1 = complete_type_list env fl2 lv1 (Mty_ident p1) fl1 in
```

Comment:
After investigation, `complete_type_list` is only ever used with package types. It might be a good idea to make it take `pack1` and `pack2` as argument.

Reply:
Done